README.md

```diff
@@ -144,13 +185,51 @@ This section describes required fields for the json configuration file.
     - **file_path**: The file path of the output video file in YUV format
 
 #### Run peerconnection_serverless
-
-To better demonstrate the usage of peerconnection_serverless, we provide an all-inclusive corpus in `examples/peerconnection/serverless/corpus`. You can use the following commands to execute a tiny example. After these commands terminates, you will get `outvideo.yuv` and `outaudio.wav`.
-
-``` shell
-sudo docker run -d --rm -v `pwd`/examples/peerconnection/serverless/corpus:/app -w /app --name alphartc alphartc peerconnection_serverless receiver.json
-sudo docker exec alphartc peerconnection_serverless sender.json
-```
+- Dockerized environment
+
+    To better demonstrate the usage of peerconnection_serverless, we provide an all-inclusive corpus in `examples/peerconnection/serverless/corpus`. You can use the following commands to execute a tiny example. After these commands terminates, you will get `outvideo.yuv` and `outaudio.wav`.
+    
+    ``` shell
+    sudo docker run -d --rm -v `pwd`/examples/peerconnection/serverless/corpus:/app -w /app --name alphartc alphartc peerconnection_serverless receiver.json
+    sudo docker exec alphartc peerconnection_serverless sender.json
+    ```
+
+- Bare metal
+
+    If you compiled your own binary, you can also run it on your bare-metal machine.
+    
+    - Linux users:
+        1. Copy the provided corpus to a new directory
+            ```shell
```

Comment:
Add empty line before ```code```. You can install https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one on VSCode to check some Markdown warning.